packages/bierzo-wallet/src/logic/account.ts

```diff
@@ -0,0 +1,30 @@
+import { Address, ChainId } from '@iov/bcp';
+import { BnsConnection } from '@iov/bns';
+
+import { getConfig } from '../config';
+import { getConnectionFor, isBnsSpec } from './connection';
+
+// getAddressByName returns the address associated with the name, or undefined if not registered
+// the name should not have the "*iov" suffix
```

Comment:
Can you use doc comments? (starting with `/**` ending with `/*`. Then you get this documentation in the IDE when you mouseover symbols